### PR TITLE
Refresh: Homepage strings (#15306)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Convert homepage strings to ftl identifiers.

## Issue / Bugzilla link

 #15306

## Testing

_Note: I have not used the gallery strings because I wanted to avoid a conflict with #15481_

### CODE REVIEW

File: bedrock/mozorg/views.py
Comment (stephaniehobson) [inline bedrock/mozorg/views.py:134]: A few lines down this is missing the code necessary to active this homepage if it is translated in languages other than English.

File: l10n/en/mozorg/home-m24.ftl
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:72]:
```suggestion
m24-home-topic-ps = Privacy & Security
```

File: l10n/en/mozorg/home-m24.ftl
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:84]: **Question (non-blocking):** Is it worth making `Rise25` a brand name variable at this point?

File: l10n/en/mozorg/home-m24.ftl
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:94]: **Question (non-blocking):** Should `IRL Podcast` also be a brand name?

**Review by alexgibson — CHANGES_REQUESTED** [review]
A few nits/suggestions and a couple of missing strings, but otherwise everything looks good. 

We should probably hold off merging this until we include the gallery strings, so that we can avoid showing mixed content for pages that get translated quickly.

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:55]: ✅ I also updated the navigation to use the brand term

File: bedrock/mozorg/templates/mozorg/home/includes/m24/media-springboard.html
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/media-springboard.html:58]: ✅ I'll make it a brand term.

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:56]: ✅

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:72]: ✅

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:84]: ✅ I'll make it a brand term.

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:94]: ✅ I'll make it a brand term.

File: bedrock/mozorg/templates/mozorg/home/home-m24.html
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/home-m24.html:31]: ✅

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:22]: ✅

**Comment by stephaniehobson** [comment]
Rebased and ready for re-review. The gallery strings are now in use.

File: l10n/en/mozorg/home-m24.ftl
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:87]: **Suggestion (non-blocking):** The term `docuseries` is common online today, but I'm not sure if it's recognized widely as a real word? (my spell checker doesn't seem to recognize it at least). 

Is it worth add an L10n comment to say it's short for "documentary series", or just using the two words for a simplified English string?

File: l10n/en/mozorg/home-m24.ftl
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:31]: **Suggestion: (non-blocking)** this one could be another good contender for en en-US only string, as I'm not entirely sure other languages will get this joke well enough to translate it?

File: l10n/en/mozorg/home-m24.ftl
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:61]: **Question:** is Goldilocks well known throughout the world - will everyone get the joke? (I would assume yes, but I feel like I am also probably ignorant here).

**Review by alexgibson — APPROVED** [review]
We need to remember to include a config here for Pontoon (also for the about page): https://github.com/mozilla-l10n/www-l10n/pull/454#issuecomment-2503198462

I spotted a couple more strings where I question if they will translate well enough, and may benefit from a simplified English version. But I will leave this up to you.

r+wc

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:31]: ✅

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:61]: ✅ It's hard to fight the urge to add the Oxford comma while I'm in here...

File: l10n/en/mozorg/home-m24.ftl
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:87]: ✅

File: l10n/en/mozorg/home-m24.ftl
Comment (janbrasna) [reply l10n/en/mozorg/home-m24.ftl]: So the jokes are effectively kept out of en-CA & en-GB this way, only good enough for en-US, right? ;) 

Nah, sorry, kidding aside — I wanted to ask about the string id rule here — when the `en` vs. `en-US` content differs: should the ftl id be made up of the fancier en-US text, or as the simplified English is used as a base for translations it should match the easy string? In this case:
- `m24-home-go-chaosfree-with` = Simplify your life with one app for all your emails…
or
- `m24-home-simplify-your-life` = Simplify your life with one app for all your emails…

(or it doesn't really matter in what locale the text matches?)

File: l10n/en/mozorg/home-m24.ftl
Comment (janbrasna) [reply l10n/en/mozorg/home-m24.ftl]: BTW aren't all these newer brands a good candidate for `fluent_linter` additions?

https://github.com/mozilla/bedrock/blob/5cf0a9091c8d088bfae84f11a05f99be84d2fb90/.github/l10n/linter_config.yml#L22-L27

**Pull request merged**
